main.nf

```diff
@@ -70,6 +70,15 @@ def helpMessage() {
         --pon                       panel-of-normals VCF (bgzipped, indexed). See: https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.php
         --pon_index                 index of pon panel-of-normals VCF
 
+    Trimming:
+        --trimFastq                   Run Trim Galore
+        --clip_r1 [int]               Instructs Trim Galore to remove bp from the 5' end of read 1 (or single-end reads)
+        --clip_r2 [int]               Instructs Trim Galore to remove bp from the 5' end of read 2 (paired-end reads only)
+        --three_prime_clip_r1 [int]   Instructs Trim Galore to remove bp from the 3' end of read 1 AFTER adapter/quality trimming has been performed
+        --three_prime_clip_r2 [int]   Instructs Trim Galore to remove bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed
+        --trim_nextseq [int]          Instructs Trim Galore to apply the --nextseq=X option, to trim based on quality after removing poly-G tails
+        --saveTrimmed                 Save trimmed FastQ file intermediates
```

Comment:
```suggestion
        --save trimmed                Save trimmed FastQ file intermediates
```

Reply:
shouldn't 
--save trimmed 
be:
--save_trimmed

Reply:
True, I must have made a typo there